### DISCUSSION TITLE
adding features to enable Google Translate and DeepL Translation Services. as well as zh_CN Wikipedia lookup

### DISCUSSION THREAD

**Discussion by forgotmymilk** [xiaoyifang/goldendict-ng#1012] [Q&A]
It appears that the current features to do these things don't work (properly). Does anyone know how to make them work? If the software does not actually support them, does the author have a plan to implement them in the near future?

**Comment by xiaoyifang** [comment]
https://github.com/xinebf/google-translate-for-goldendict

deepl can be added through program dictionary.
https://pypi.org/project/deepl/#:~:text=python3%20%2Dm%20deepl%20%2D%2Dauth%2Dkey%3DYOUR_AUTH_KEY%20text%20%2D%2Dto%3DDE%20%22Text%20to%20be%20translated.%22


change `AUTH_KEY` to your own key.
configure it with program
like 
`python3 -m deepl --auth-key=YOUR_AUTH_KEY text --to=DE %GDWORD%`

![image](https://github.com/xiaoyifang/goldendict-ng/assets/105986/93ef5657-eb78-4e9e-8170-dcb1607f84b6)
